spec/feature/user_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'user'
+require_relative 'database_helpers'
+
+describe User do
+  describe '.create' do
+    it 'creates a new user' do
```

Comment:
Well written tests over all. 
I'm unsure why this file is in the feature folder.